client/Components/GameBoard/Droppable.jsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { DropTarget } from 'react-dnd';
+import classNames from 'classnames';
+
+import { ItemTypes } from '../../constants';
+
+const validTargets = {
+    hand: [
+        'play area',
+        'discard pile',
+        'draw deck',
+        'dead pile',
+        'out of game',
+        'conclave'
+    ],
+    'play area': [
+        'discard pile',
+        'hand',
+        'draw deck',
+        'dead pile',
+        'out of game',
```

Comment:
It should be noted (but probably not worth fixing) that this and the other out-of-game drag combinations can lead to invalid locations. Example: dragging a Rains Scheme plot from out-of-game to the play area, or an out-of-game character to the plot deck.

This will already be prevented on the server-side implementation for drag-drop, so it won't lead to invalid game states. It just may lead to the UI displaying a drag as okay and then not following through with it.

Reply:
It does, and also things like dragging attachments to the dead pile.  It's only intended as a rudimentary guide.  It could be tightened up to be 100% accurate, but it makes it a lot more complex

Reply:
Yeah. Like I said, worth acknowledging, but probably not really worth fixing.